week1/homework/README.md

```diff
@@ -28,76 +93,68 @@ node .
 ## Assignment
 
 Create an HTTP server that can add and subtract from a number, which we will
-call `state`. Use project in `week1` directory as starting material.
-Pay extra attention to line 21, which contains some hints for this week
-`console.log('New http request received', request.url);`
+call `state`. Use project in `week1/lecture` directory for reference.
+
```

Comment:
Your modifications should be limited to the `src` folder. Do not make any changes to the `test` folder.

Reply:
Make sense. Should have added that.